app/views/application/_navbar.html.erb

```diff
@@ -0,0 +1,9 @@
+<nav>
+<h3><%= link_to t("application.home"), root_path %> |
+<% if current_user %>
+  <%= link_to t("application.signout"), session_path(current_user), method: :delete %> |
+<% else %>
+  <%= link_to t("application.signup"), new_user_path %> |
+  <%= link_to t("application.signin"), new_session_path %></h3>
+<% end %>
```

Comment:
If you wish not to have the links in a list, then I would at least remove them from being nested in a `h3` because semantically `h3` is a heading level, and here it is used for styling purposes.

But really up to you.

Reply:
The styling isn't final, I will do it in a PR when it is